packages/pharaoh/lib/src/http/request.dart

```diff
@@ -71,12 +71,13 @@ class RequestContext {
 class Request extends Message<dynamic> implements $Request<dynamic> {
   final HttpRequest _req;
   final Map<String, dynamic> _params = {};
+  final Map<String, dynamic> _query = {};
```

Comment:
We don't need this new declaration. Because we'll never attempt modifying the request query parameters. 

So it's safe to just pass the value directly on the getter call like so 
```dart
Map<String, dynamic> get query => _req.uri.queryParameters
```

Reply:
Alright, got it!